index.html

```diff
@@ -2,7 +2,43 @@
 <html>
   <head>
     <title>Delicious: Chocolate Pizza</title>
-    <link rel="stylesheet" type="text/css" href="css/reset.css">
-    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" type="text/css" href="css/reset.css" />
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
   </head>
+  <body>
+    <header>
+      <img src="img/logo.png" alt="Delicious - company logo" />
+      <section id="company-name"></section>
+        <h2>Delicious</h2>
+        <h5>THE BEST FOOD ON THE WEB.</h3>
+        <ul>
+          <li><img src="img/fb-icon.png" alt="" /></li>
+          <li><img src="img/twit-icon.png" alt="" /></li>
+          <li><img src="img/gp-icon.png" alt="" /></li>
+          <li><img src="img/insta-icon.png" alt="" /></li>
+          <li><img src="img/flic-icon.png" alt="" /></li>
+          <li><img src="img/pint-icon.png" alt="" /></li>
+          <li><img src="img/rss-icon.png" alt="" /></li>
+          <li><img src="img/mail-icon.png" alt="" /></li>
+        </ul>
+      </section>
+    </header>
+    <main>
+    </main>
+    <article>
+      <img src="img/van-pic.png" alt="Venessa Stevenson headshot" />
+      <div>
+        <h3>Venessa Stevenson</h3>
+        <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
+      </div>
+      <button type="button">SHARE RECIPE</button>
+    </article>
+    <footer>
+      <ul>
+        <li>Delicious &copy; 2013</li>
```

Comment:
I probably wouldn't have chosen to make these two separate `li`s; they're going to be on a single line as a single piece of text eventually, so I'd just make them one piece of text.